wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -15,16 +15,180 @@
 
 <main id="main" class="site-main">
 
+<?php
+if ( '' === get_query_var( 'search' ) && empty( $_GET ) && is_post_type_archive() ) :
+	?>
+	<section>
+		<div class="row align-middle between section-heading section-heading--with-space large-space">
+			<h1 class="section-heading_title bigger"><?php esc_html_e( 'Lesson Plans', 'wporg-learn' );?></h1>
+			<a href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) . '?_view=all' ); ?>" class="button button-large button-secondary">
+				<?php esc_html_e( 'Browse all lesson plans', 'wporg-learn' ); ?>
+			</a>
+		</div>
+
+		<hr>
+
+		<div class="row align-middle between section-plan_description">
+			<div class="section-plan_text">
+				<?php esc_html_e( 'Want to help others learn about WordPress? Read through, use, and remix these lesson plans.', 'wporg-learn' );?>
+			</div>
+			<?php
+			set_query_var( 'post_type', 'lesson-plan' );
+			get_template_part( 'template-parts/component', 'archive-search' );
+			?>
+		</div>
+
+		<hr>
+		<?php
+		$categories = get_terms( 'wporg_lesson_category', array(
+			'hide_empty' => false,
+			'orderby'    => 'id',
+		) );
+		?>
+		<div class="row lesson-plan-taxonomy categories">
+			<div class="card-grid card-grid_4">
+				<h2 class="h4 lesson-plan-taxonomy-header"><?php echo esc_html__( 'Topic', 'wporg-learn' ); ?></h2>
+				<div class="lesson-plan-taxonomy-description"><?php echo esc_html__( 'Browse lesson plans by their high-level topic.', 'wporg-learn' ); ?></div>
+				<?php foreach ( $categories as $category ) :
+					$is_sticky = get_term_meta( $category->term_id, 'sticky', true );
+					if ( $is_sticky ) :
+						?>
+				<div class="card">
+					<div class="icon">
+						<?php
+						$category_icon = get_term_meta( $category->term_id, 'dashicon-class', true ) ? get_term_meta( $category->term_id, 'dashicon-class', true ) : 'wordpress-alt';
+						?>
+						<a href="<?php echo esc_url( get_term_link( $category ) ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $category_icon ); ?>"></span></a>
+					</div>
+					<p class="taxonomy-title"><a href="<?php echo esc_url( get_term_link( $category ) ); ?>"><?php echo esc_html( $category->name ); ?></a></p>
```

Comment:
The 2 anchors within the card affect the keyboard accessibility of these sections. It takes 2 tab presses to move from card to card. I think you could wrap both the title and icon in one anchor.

Reply:
Running an accessibility audit using Chrome dev tools Lighthouse shows that the icon anchors don't have names for a screen reader. This would also be fixed by wrapping both the icon and title I believe.